lm_eval/tasks/med_concepts_qa/README.md

```diff
@@ -32,16 +32,16 @@ Our benchmark serves as a valuable resource for evaluating the understanding and
 #### Tasks
 
 
-* `ICD9-CM` - ICD9-CM (diagnosis codes, ICD9 format) question-answering. This involves providing information, clarifications, and answering questions related to ICD-9-CM (International Classification of Diseases, 9th Revision, Clinical Modification) diagnosis codes.
+* `mcqa_icd9cm` - ICD9-CM (diagnosis codes, ICD9 format) question-answering. This involves providing information, clarifications, and answering questions related to ICD-9-CM (International Classification of Diseases, 9th Revision, Clinical Modification) diagnosis codes.
```

Comment:
`medconceptsqa` or `med_concepts_qa` ? 

MCQA = multiple choice QA frequently

Reply:
I prefer `med_concepts_qa` instead of MCQA. I have change it.